distrobox-create

```diff
@@ -547,7 +547,6 @@ generate_command() {
 		--label \"manager=distrobox\"
 		--env \"SHELL=$(basename "${SHELL:-"/bin/bash"}")\"
 		--env \"HOME=${container_user_home}\"
-		--mount type=tmpfs,destination=/run
```

Comment:
Here I'd add `--env \"container=${container_manager}\"`
So that also on docker we will have the `container=docker` env variable like on podman we have `container=podman`